helm-chart/binderhub/templates/deployment.yaml

```
@@ -61,6 +61,12 @@ spec:
       {{- with .Values.extraVolumes }}
       {{- . | toYaml | nindent 6 }}
       {{- end }}
+      {{- with .Values.image.pullSecrets }}
+      imagePullSecrets:
+        {{- range . }}
+        - name: {{ . }}
+        {{- end }}
+      {{- end }}
```

I think this is now possible, allowing a binderhub user to rely on jupyterhub chart wide configuration options like below, so its only needed to be configured once. Alternatively, by specifying something special just under `image.pullSecrets`.

```yaml
jupyterhub:
  imagePullSecret: ...
  imagePullSecrets: ...
```

```suggestion
      {{- with include "jupyterhub.imagePullSecrets" (dict "root" . "image" .Values.image) }}
      imagePullSecrets: {{ . }}
      {{- end }}
```